scylla/tests/common/utils.rs

```diff
@@ -300,3 +300,31 @@ impl PerformDDL for CachingSession {
         self.execute_unpaged(query, &[]).await.map(|_| ())
     }
 }
+
+#[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
I don't understand why I had to add this line here even though I use this struct in my test. Should it not be not a dead_code because of that?